src/pages/Auth/Login/index.tsx

```diff
@@ -54,31 +71,77 @@ export default function Login(): JSX.Element {
       isAutoLoginChecked: false,
     },
   });
-  const submitLogin = useLoginRequest();
+
+  const [error, setError] = useState<boolean>(false);
+  const submitLogin = useLoginRequest({ onError: setError });
 
   return (
     <div className={styles.template}>
       <div className={styles.content}>
         <AuthTitle />
         <div className={styles.form}>
-          <form className={styles.loginform} onSubmit={handleSubmit(submitLogin)}>
+          <form
+            className={styles.loginform}
+            onSubmit={handleSubmit(submitLogin)}
+          >
+            <div className={styles.error}>
+              { error && (
+                <>
+                  <ErrorIcon
+                    aria-hidden
+                  />
```

Comment:
통일되지 않은 띄어쓰기와, 가독성을 해칠 수 있는 불필요한 줄바꿈이라고 생각해요.
```suggestion
              {error && (
                <>
                  <ErrorIcon aria-hidden />
```

Reply:
수정했습니다. [48c82af](https://github.com/BCSDLab/JJBAKSA_FRONT_END/pull/43/commits/48c82afba5c6f5ea99b127595f871316215cbfec)